src/frontEnd/components/containers/App.js

```diff
@@ -1,22 +0,0 @@
-import {connect} from 'react-redux';
-import App from '../App.jsx';
-import {ChangeCurrentView,isLoggedIn,getRooms}from '../../actions.js';
-
-const mapStateToProps = (state)=>{
-  return{
```

Comment:
In ES6, fat arrow with parentheses is an implicit return, so this can just be written as
(state) => (
{
currentView: state.currentView, ... }) without need for '{return...}'. This is the case for all functions, but can be most immediately applied to all mapStateToProps and mapDispatchToProps.